integration-tests/rita.py

```diff
@@ -42,7 +42,7 @@
 COMPAT_LAYOUT = os.getenv('COMPAT_LAYOUT', None)
 
 BACKOFF_FACTOR = float(os.getenv('BACKOFF_FACTOR', 1))
-CONVERGENCE_DELAY = float(os.getenv('CONVERGENCE_DELAY', 50))
+CONVERGENCE_DELAY = float(os.getenv('CONVERGENCE_DELAY', 100))
```

Comment:
We don't need to do this, as this value is carefully tuned to make the CI tests run faster. If you want it to be slower for you you can set the environmental var CONVERGENCE_DELAY=100 ./rita.sh